keyboards/zos/65s/info.json

```diff
@@ -0,0 +1,175 @@
+{
+    "manufacturer": "ZOS Keyboards",
+    "keyboard_name": "ZOS 65s",
+    "maintainer": "Zenn",
+    "bootloader": "atmel-dfu",
+    "diode_direction": "COL2ROW",
+    "features": {
+        "bootmagic": true,
+        "command": false,
+        "console": false,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": true
+    },
+    "matrix_pins": {
+        "cols": ["D2", "D1", "D0", "D7", "B4", "B5", "F0", "B6", "C6", "F1", "C7", "F4", "F5", "F6", "F7"],
+        "rows": ["D5", "D4", "D6", "D3", "B7"]
+    },
+    "processor": "atmega32u4",
+    "url": "https://www.zoskb.com",
+    "usb": {
+        "device_version": "1.0.0",
+        "pid": "0x0001",
+        "vid": "0x25D5"
```

Comment:
This vendor ID is already in use. 

https://www.the-sz.com/products/usbid/index.php?v=0x25D5&p=&n=

The further past 0x4000, the better, as the 0x3xxx range is actively being filled

Reply:
Thanks for the note, I have updated the Vendor ID.